### ISSUE TITLE
Reintroduce a single variable param -p for projects on  "dotnet run"

### ISSUE THREAD

**Issue by jannickj** [dotnet/sdk#22792]
I just installed dotnet 6 and was met with 

"Warning NETSDK1174: The abbreviation of -p for --project is deprecated. Please use --project."

`dotnet run -p` is single command I write the most all day at my work, lengthening it to `--project` is a very annoying change since it just makes dotnet more cumbersome and overall less enjoyable.. I understand I can setup aliases etc but then you run into the issue of having those aliases setup everywhere. This is the same reason people eventually default to bash because those are tools that are always available.

So in short going from `-p` -> `--project` might seem like an innocent change, but you are playing with one of the most used commands, and not having 1 letter params for it just seems like someone -who didn't know the impact- just made this change.

plz un-deprecate it!

Update: I just found out it's even worse, the warning is being printed on stdout and not stderr so `dotnet run -p [project] | jq` doesn't work anymore, meaning effectively `--project` is a hard requirement (for cli usage).

**Comment by dusda** [comment]
It's been five years, now, are you ever going to actually remove `-p` or just leave this annoying warning? Also it's still printing to stdout instead of stderr. Why not just let it behave like `dotnet run src/Bleh.Web`?
